main.py

```diff
@@ -1,36 +1,43 @@
 from utils import Utils
+from move_robot import MoveRobot
+
 NUMBER_OF_PARTS = 4
 BIG_GRIPPER = 0
 SUCTION = 1
 SMALL_GRIPPER = 2
 FINAL_COVER = 3
 
+
 class Controller():
     def __init__(self):
         self.in_zero_position = False
         self.part_id = None
         self.colour_id = None
         self.location = None
         self.orientation = None
+        self.fixture_x = 20
+        self.fixture_y = 15

```

Comment:
Define "magic numbers" as constants

Reply:
Dont know if this was just for testing? In that case I guess it will be changed later.